### PR TITLE
Master development 929

### LINKED ISSUE #929: Master development
Use the script from dedis/Coding to create automatically a new stable branch. Hoping this will help to keep `v1.1` and `master` better in sync...

### PR DESCRIPTION
**Pull request by ineiti**
Each step is in its own commit:
- skipchain-backports and fixing of tests
- adding `// +build experimental`
- short fix
- adding stable-scripts

Closes #929

### CODE REVIEW

File: stable/overwrite/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/mattn/goveralls
+
+go_import_path: gopkg.in/dedis/cothority.v1
+
+script:
+  - make test
+
+after_success:
+  - goveralls -coverprofile=profile.cov -service=travis-ci -repotoken $COVERALLS_TOKEN
```
Comment (nikkolasg) [inline stable/overwrite/.travis.yml]: Still not sure we need that ! See the [readme](https://github.com/mattn/goveralls#github-integration) for the simple case.

Comment (ineiti) [reply stable/overwrite/.travis.yml]: I removed the variable but not the reference - I'm quite surprised travis passed anyway...

**Comment by nikkolasg** [comment]
Is it worth mentionning the docker thingie ?

**Comment by ineiti** [comment]
I don't understand your comment.

**Comment by ineiti** [comment]
then go and merge!

**Pull request merged**